### ISSUE TITLE
[Telemetry] Logins

### ISSUE THREAD

**Issue by liuche** [mozilla-mobile/fenix#5586]
#### Description & Product Manager / Data Scientist User Story
- As a product owner, I want to know how users interact with features related to logins and what features they find most useful.

#### What questions will you answer with this data?
- How often users let their browser save their logins?
- How often do users manage their saved logins?
- What login management features are used most frequently?
- What login management features are not used often (so we can determine whether we need to remove or enhance them)

#### What probes (suggested, if applicable)
- [ ] choose whether Fenix will remember logins
- [ ] sync logins
- [ ] view Logins in settings
- [ ] open an individual login
- [ ] copy a login
- [ ] view a password
- [ ] User says yes/no to saving a login on a site

### Acceptance Criteria (Added by PM)
- Event pings can be queried via re:dash
- We are sending telemetry events for the actions listed in the requirements
- We have documented the telemetry
- We have asked a data steward to [review](https://github.com/mozilla/data-review/blob/master/request.md) the telemetry
- NOT an AC: Data science to create dashboard or further graphs (this will be a separate issue, this issue is only about hooking up the events described and that they can be queried in re-dash)

**Comment by ekager** [comment]
As we are adding features we will add remaining telemetry see the checklist for remaining work

**Comment by ekager** [comment]
We already have telemetry for "sync_now" and opening sync settings. Does "sync logins" mean a new probe for which items in CWTS settings are selected/changed?

**Comment by vesta0** [comment]
@ekager Yes that is right. That just made me realize that we probably don't collect telemetry for Sync Bookmarks and Sync history separately either. We only recently shipped the ability for user to interact with individual sync options. I created a ticket for those ones: #6396

**Comment by AndiAJ** [comment]
Hi, I've just checked this on the latest **Nightly Build Nightly Build #13240608 from 20/11** using a Google Pixel 3a XL (Android 9).

 ❌ Sync logins - No ping is generated
✔️ Access Logins in settings Ping **4205dcb5-5c49-4a9c-b2ee-cbdabaa27dcb**
✔️ Open an individual login Ping **2ce4543d-5ec0-42ac-addb-674bb1a30ffa**
 ❌ Copy a login - The ping is generated **only when you copy the password** Ping **7e303500-1403-49ad-9630-281e878a5be0** 
✔️ View a password Ping **e7f8663a-63c9-481e-80e6-0496b56cc567**
 ❌ Choose whether Fenix will remember logins - **Blocked** by #5545 
 ❌ Search logins - **Blocked** by #5774 
 ❌ Delete a login  - **Blocked** by #5774  

[Logcat](https://github.com/mozilla-mobile/fenix/files/3869958/loggins.txt)
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/aj-t201100)

@ekager & @liuche How should we proceed with this matter taking into account that some pings aren't generated and some are blocked ? :relaxed:

**Comment by AndiAJ** [comment]
Hi, I've just checked this on the latest **Nightly Build #13310607 from 11/27** using a Google Pixel 3a (Android 9).

❌ Sync logins - No ping is generated
❌ Access Logins in settings - **Blocked** by #6628 
❌ Open an individual login  - **Blocked** by #6628
❌ Copy a login - **Blocked** by #6628 
❌ View a password Ping  - **Blocked** by #6628
❌ Choose whether Fenix will remember logins - **Blocked** by #5545 
❌ Search logins - **Blocked** by #5774 
❌ Delete a login  - **Blocked** by #5774

**Comment by AndiAJ** [comment]
Hi, re-checked this issues on the latest **Nightly Build #13440604 from 12/10** using a Google Pixel 3a (Android 9)

❌ Sync logins - No event type ping is generated
✔️ Turn ON Logins sync - Ping **ddb1218d-a68a-4385-b61f-1e00afb14c98**
✔️ Turn OFF Logins sync - Ping **c570c5e9-f192-403d-b8ef-d2635b348b0a**
✔️ Access Logins in settings - Ping **77d3dd7d-7212-47b5-acf7-75e836656dfc**
✔️ Open an individual login - Ping **c4f8e513-b08f-4091-b00b-586aa4076f44**
✔️ Copy a login (Site, Username, Password) - Ping **5583a6e6-31c2-4dd1-8d3d-fe4da74d6895**
✔️ View a password Ping - Ping **c7d86d64-c383-4e2f-9bd4-de400779d994**
❌ Choose whether Fenix will remember logins - **Blocked** by #5545 
❌ Search logins - **Blocked** by #5774
❌ Delete a login  - **Blocked** by #5774  

[Logcat](https://github.com/mozilla-mobile/fenix/files/3945646/Loggg.txt)
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/aj-test1012001)

@ekager - Please review and let me know if I missed something while checking, also please advise how should we proceed with this ticket considering that some features aren't implemented yet.  :relaxed:

**I'll remove the QA needed label until further advice.**

**Comment by ekager** [comment]
@AndiAJ 

> -  Sync logins - No event type ping is generated
This should generate a general "sync now" event type when we sync. @vesta0 do you need to know which categories are synced every time a user generates a sync event? We are already recorded when these categories are changed.


> - Choose whether Fenix will remember logins - Blocked by #5545

This should be ready to test! Thank you!

**Comment by vesta0** [comment]
This should be good enough for now @ekager thank you :)

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20280607 from 1/28`** using a OnePlus A3 (Android 6.0.1).

✔️ Sync logins - Ping **7e2644fd-f54c-413b-bcee-e2a3823a18ce**
✔️ Turn ON Logins sync - Ping **8c753782-2451-4010-b59f-13f4533e7fce**
✔️ Turn OFF Logins sync - Ping **c072ca20-7297-43f4-a9f0-aa80bc7895f3**
✔️ Access Logins in settings - Ping **858a6ceb-c265-4e7d-8357-470c6835560a**
✔️ Open an individual login - Ping **e929b69c-00e9-41ed-a5af-f0806608a397**
✔️ Copy a login (Site, Username, Password) - Ping **02f187cd-9338-4828-8718-73e7e01cbe17**
✔️ View a password Ping - Ping **be8365e9-708b-437e-9c1e-446864614bee**
✔️ Choose whether Fenix will remember logins :
NEVER_SAVE -  Ping  **29aaaca9-4c3a-4e73-b57c-f4fb76ea1102**
ASK_TO_SAVE - Ping **b4002edf-0474-4d17-97ff-5e48cb23a0d3**

[Logcat](https://github.com/mozilla-mobile/fenix/files/4123093/Log.txt)
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/aj-test2801000)
